src/index.ts

```diff
@@ -167,20 +169,35 @@ function barfTurn() {
     // d. get dressed
     meatOutfit(embezzlerUp, [], underwater);
 
-    adventureMacroAuto(
-      location,
-      Macro.externalIf(
-        underwater,
+    if (
```

Comment:
Do we want to do this even if nobarf is true?

Reply:
Would you ever run `nobarf` alongside `ascend` while also overdrunk?